frontend/packages/component-library/src/actionBlock/ActionBlock.tsx

```diff
@@ -65,7 +65,6 @@ export const getButtons = (
     <div className={moduleStyles.buttonWrapper}>
       {primaryButton && (
         <LinkButton
-          {...primaryButton}
```

Comment:
Realized I never actually deleted this in [this PR](https://github.com/code-dot-org/code-dot-org/pull/64639#discussion_r2010108143) 🤦‍♀️